portfolio/src/main/java/com/google/sps/data/UserInfo.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.data;
+
+public class UserInfo {
+  private String email;
+  private boolean showEmail;
+  private String username;
```

Comment:
Can be private final if only initialize once.

Reply:
Done.